rust/src/rustlibs_toast/mod.rs

```diff
@@ -0,0 +1,14 @@
+use byondapi::value::ByondValue;
+#[cfg(target_os = "windows")]
+use notify_rust::Notification;
```

Comment:
Thank you for making this a compile time check for windows so we don't need to figure out dbus hell